pkg/mutation/assign_mutator.go

```diff
@@ -39,6 +39,10 @@ func (m *AssignMutator) SchemaBindings() []SchemaBinding {
 	return m.bindings
 }
 
+func (m *AssignMutator) Value() runtime.RawExtension {
+	return m.assign.Spec.Parameters.Assign
```

Comment:
Echoing and earlier comment about this actually returning Assign.Value

Reply:
Done in previous PR (#987)